pygfx/materials/_background.py

```diff
@@ -110,51 +109,18 @@ def map(self, map):
         self._map = map
 
 
-class BackgroundSkyboxMaterial(BackgroundImageMaterial):
-    """A cube image background, resulting in a skybox.
-    Use the up property to orient the skybox.
-    """
-
-    uniform_type = dict(
-        tex_index="4xi4",
-        yscale="f4",
-    )
+class BackgroundSkyboxMaterial(Material):
+    """A cube image background, resulting in a skybox."""
 
-    def __init__(self, map=None, up=(0, 1, 0), **kwargs):
-        super().__init__(map=map, **kwargs)
-        self.up = up
+    def __init__(self, *, map=None, **kwargs):
+        super().__init__(**kwargs)
+        self.map = map
 
     @property
-    def up(self):
-        """A Vector3 defining what way is up. Can be set to e.g. the
-        controller's up vector. The given vector is "rounded" to the
-        closest vector that is fully in one dimension.
-        """
-        return self._up
-
```

Comment:
I think we still want this up property to be able to orient the skybox, e.g. if you want a skybox in a scene where z+ is up.